frontend/packages/core/src/tests/tab-context.test.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import TabContext from "@mui/lab/TabContext";
+import { render } from "@testing-library/react";
+
+import "@testing-library/jest-dom";
+
+import { Tab, Tabs } from "../tab";
+
+jest.mock("@mui/lab/TabContext", () => {
+  return jest.fn(() => null);
+});
+
+test("TabContext is called with specified value", () => {
```

Comment:
From what we talked about, lets just remove this test as it tries to test internal logic and isn't necessary for verification of functionality or the presentation layer. We'll extend the tabs test functionality after we've migrated.